src/main/java/io/obswebsocket/community/client/message/response/mediaInputs/GetMediaInputStatusResponse.java

```diff
@@ -0,0 +1,46 @@
+package io.obswebsocket.community.client.message.response.mediaInputs;
+
+import com.google.gson.annotations.SerializedName;
+import io.obswebsocket.community.client.message.request.Request;
+import io.obswebsocket.community.client.message.response.RequestResponse;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString(callSuper = true)
+public class GetMediaInputStatusResponse extends RequestResponse {
+    private Data responseData;
+
+    public GetMediaInputStatusResponse() {
+        super(Request.Type.GetMediaInputStatus);
+    }
+
+    @Getter
+    @ToString
+    public static class Data {
+        private MediaState mediaState;
+        private Long mediaDuration; // optional
+        private Long mediaTimestamp; // optional
+    }
+
+    public enum MediaState {
```

Comment:
I'm not sure if this should stay in this class or be moved to the `model` package.
@TinaTiel what do you think?

Reply:
Nah, probably fine as-is for now.
It's only in 2 responses, and they differ on whether mediaTimestamp is optional.

Reply:
I was talking about the enum there

Reply:
OH...My mistake. 
Coincidentally, my response is the same; so far MediaState is only used in this one response so it makes sense to keep it here for now.